app/src/main/res/layout/fragment_participant_flow_matching.xml

```diff
@@ -76,52 +77,59 @@
-            android:textColor="@color/colorTextOnSecondary"
-            android:visibility="@{viewModel.selectedParticipant == null ? View.GONE : View.VISIBLE}"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent" />
+            <Button
+                android:id="@+id/btn_match_participant"
+                android:layout_width="0dp"
+                android:layout_weight="1"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/ButtonTextStyling"
+                android:backgroundTint="@color/colorAccent"
+                android:text="@string/participant_matching_btn_match_selected"
+                android:textSize="@dimen/text_size_12_16"
+                android:visibility="@{viewModel.selectedParticipant == null ? View.GONE : View.VISIBLE}"
+                />
 
-        <Button
-            android:id="@+id/btn_report_adverse_effects"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="16dp"
-            android:textAppearance="@style/ButtonTextStyling"
-            android:backgroundTint="@color/colorAccent"
-            android:text="@string/participant_matching_btn_report_adverse_effects"
-            android:textColor="@color/colorTextOnSecondary"
-            android:visibility="@{viewModel.selectedParticipant == null ? View.GONE : View.VISIBLE}"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintStart_toEndOf="@+id/btn_match_participant"
-            app:layout_constraintEnd_toStartOf="@+id/btn_new_participant" />
+            <Button
+                android:id="@+id/btn_report_adverse_effects"
+                android:layout_width="0dp"
+                android:layout_weight="1"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/ButtonTextStyling"
+                android:backgroundTint="@color/colorAccent"
+                android:layout_marginHorizontal="@dimen/margin_8_16"
+                android:text="@string/participant_matching_btn_report_adverse_effects"
+                android:textSize="@dimen/text_size_12_16"
+                android:textColor="@color/colorTextOnSecondary"
+                android:visibility="@{viewModel.selectedParticipant == null ? View.GONE : View.VISIBLE}"
+                app:layout_constraintBottom_toBottomOf="parent"/>
+
+            <Button
+                android:id="@+id/btn_new_participant"
+                android:layout_width="0dp"
+                android:layout_weight="1"
+                android:layout_height="wrap_content"
+                android:textAppearance="@style/ButtonTextStyling"
+                android:backgroundTint="@color/colorAccent"
+                android:text="@string/participant_matching_btn_edit_participant"
```

Comment:
This button can be given the same size as that of `adverse effects` button.Right now it looks bigger than other buttons.